### ISSUE TITLE
[Accessibility] Warn when no alt text is attached to a post containing an image

### ISSUE THREAD

**Issue by orowith2os** [mastodon/mastodon#22050]
### Pitch

When posting a toot on Mastodon, some users will attach an image. Sometimes, there's text accompanying it, describing it for the visually impaired, and those that might not be able to understand the image (such as with fonts that are unreadable by some).

It would be useful for Mastodon to warn the user when making a post that doesn't contain any alt text on an image.

Examples:

I make a post without an image, no issue, it posts as normal.

I make a post containing an image with alt text, no issue, it posts as normal.

I make a post containing an image without alt text, it warns me about it before posting.

I make a post containing multiple images, one of which doesn't have alt text, it warns me about it before posting.

Useful alongside https://github.com/mastodon/mastodon/issues/19907

### Motivation

Some users might not remember to add alt text to their images, or might not know about the feature.

**Comment by connorshea** [comment]
Duplicate of #19974 / #7804, I've got a PR open to implement this here: #21679

**Comment by ineffyble** [comment]
Closing as duplicate.
